mev-boost-rs/tests/integration.rs

```diff
@@ -102,15 +102,23 @@ async fn test_end_to_end() {
 
     // start upstream relay
     let validator_mock_server_url = validator_mock_server.url("");
-    let relay_config =
-        RelayConfig { beacon_node_url: validator_mock_server_url, ..Default::default() };
+    // note: non-default secret key required. otherwise "no valid bids returned for proposal".
```

Comment:
want to call this out. can dig into deeper, but there was an issue when the relay had the default `SecretKey`, so I went with the one previously in the `Relay` constructor

Reply:
appears that signature does not verify because `SecretKey::default` gives `PublicKey` with BLST error [`BLST_PK_IS_INFINITY`](https://github.com/ralexstokes/ethereum-consensus/blob/46ca1716f2b1651e59b8a565b5bbcadf6b052fa5/src/crypto.rs#L55) on [`validate_bid`](https://github.com/ralexstokes/mev-rs/blob/0332af04d0777625aea5e1be3608d4aef341d505/mev-boost-rs/src/relay_mux.rs#L125):

```
invalid signed builder bid: blst error: public key is infinity for bid
```

I will add a more specific note there

Reply:
yeah im fine if we just avoid this as the default for now